## mysql-missing-include.diff by Jens Peter Secher <jps@debian.org>
##
## Include missing declaration of free().

Index: neko/libs/mysql/my_proto/my_proto.c
===================================================================
--- neko.orig/libs/mysql/my_proto/my_proto.c	2009-07-28 22:54:44.000000000 +0200
+++ neko/libs/mysql/my_proto/my_proto.c	2009-07-28 22:55:13.000000000 +0200
@@ -20,6 +20,7 @@
 /* ************************************************************************ */
 #include <math.h>
 #include <string.h>
+#include "stdlib.h"
 #include "my_proto.h"
 
 #define MAX_PACKET_LENGTH 0xFFFFFF
